src/Cmfd.cpp

```diff
@@ -10,7 +10,7 @@
 Cmfd::Cmfd() {
 
   /* Initialize Geometry and Mesh-related attribute */
-  _polar_quad = NULL;
+  _quadrature = NULL;
```

Comment:
This attribute is called `_quad` in `Solver` and `_quadrature` in `Cmfd`. Perhaps they should use consistent notation since they are one and the same? I personally prefer the more verbose `_quadrature` but it is up to you.